.github/workflows/integrationTest.yml

```diff
@@ -971,4 +982,80 @@ jobs:
 #          max_attempts: 3
 #          timeout_minutes: 8
 #          retry_wait_seconds: 5
-#          command: cd terraform/ec2/linux && terraform destroy --auto-approve
\ No newline at end of file
+#          command: cd terraform/ec2/linux && terraform destroy --auto-approve
+
+  EC2AcceptanceIntegrationTest:
+    needs: [ECSFargateIntegrationTest, ECSEC2IntegrationTest, EC2LinuxIntegrationTest, GenerateTestMatrix]
+    name: 'Acceptance'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        arrays: ${{ fromJson(needs.GenerateTestMatrix.outputs.ec2_acceptance_matrix) }}
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: ${{env.CWA_GITHUB_TEST_REPO_NAME}}
+          ref: ${{env.CWA_GITHUB_TEST_REPO_BRANCH}}
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ env.TERRAFORM_AWS_ASSUME_ROLE }}
+          aws-region: us-west-2
+
+#      - name: Cache if success
```

Comment:
Why would we disable the cache?

Reply:
I might have done this for my development purposes because with cache, it probably didn't re-run when I had new commit in the test pkg. It can go imo for the merge